build/ship.sh

```diff
@@ -47,7 +47,7 @@ npm run release
 printf "\n%s=======================================================%s" "$magenta" "$end"
 printf "\n%sCopy it over...%s" "$magenta" "$end"
 printf "\n%s=======================================================\n%s" "$magenta" "$end"
-cp -rf _gh_pages/. ../bs-docs/
+cp -rf _site/. ../bs-docs/
```

Comment:
@mdo this reminds me that we should probably nuke this script if you don't use it. I don't use it, but if you do we can keep it.

Reply:
Nuke it, I don't use it.

Reply:
Cool, I'll do it in a separate PR later.